prime-router/src/main/kotlin/azure/LookupTableFunctions.kt

```diff
@@ -224,23 +270,42 @@ class LookupTableFunctions(
         @BindingName("tableName") tableName: String,
         @BindingName("tableVersion") tableVersion: Int
     ): HttpResponseMessage {
-        return getOktaAuthenticator(PrincipalLevel.SYSTEM_ADMIN).checkAccess(request) {
-            try {
-                if (!lookupTableAccess.doesTableExist(tableName, tableVersion))
-                    HttpUtilities.notFoundResponse(
-                        request,
-                        "Table $tableName with version $tableVersion does not exist."
-                    )
-                else {
-                    lookupTableAccess.activateTable(tableName, tableVersion)
-                    val json = mapper
-                        .writeValueAsString(lookupTableAccess.fetchVersionInfo(tableName, tableVersion))
-                    HttpUtilities.okResponse(request, json)
-                }
-            } catch (e: Exception) {
-                logger.error("Unable to activate lookup table $tableName", e)
-                HttpUtilities.internalErrorResponse(request)
+        // Do authentication
+        val claims = AuthenticatedClaims.authenticate(request)
+            ?: return HttpUtilities.unauthorizedResponse(request, authenticationFailure)
+        // Do authorization
+        if (!authorizedForLookupWrite(claims)) {
+            return HttpUtilities.unauthorizedResponse(request, authorizationFailure)
+        }
+        logger.info("User ${claims.userName} is authorized for endpoint ${request.uri}")
```

Comment:
Remove or change to debug

Reply:
Keeping auth related logging as 'info' so it remains in the logs.